Monika After Story/game/zz_selector.rpy

```diff
@@ -79,6 +79,10 @@ init -100 python in mas_selspr:
             "change": "Can you tie your hair with something else?",
             "wear": "Can you tie your hair with something else?",
         },
+        "_ev": "monika_earrings_select",
+        "_min-items": 1,
+        "change": "Can you change your earrings?",
+        "wear": "Can you wear earrings?",
```

Comment:
```suggestion
        "earrings": {
            "_ev": "monika_earrings_select",
            "_min-items": 1,
            "change": "Can you change your earrings?",
            "wear": "Can you wear earrings?",
        },
```

can you also move this up so its in alphabetical order with the rest of the ACS?